### ISSUE TITLE
jQuery not found

### ISSUE THREAD

**Issue by Kein1945** [Kikobeats/uno-zen#66]
Hi. I'm newbie with gulp/bower, but when i've tried to install your theme, I catched exception "jQuery not found".
I fix it with adding line to gulpfile.coffee:

```
   vendor : ['assets/vendor/jquery/dist/jquery.js'
              'assets/vendor/fastclick/lib/fastclick.js'
```

I don't know it's correct way adding dist path or not. But this fix my problem.
Or may be I doing something wrong and catch this error?

**Comment by Kikobeats** [comment]
- Ghost 0.6 include jquery.
- Ghost 0.7 not include jquery.

Give me hours to update the theme :-)

**Comment by Kikobeats** [comment]
will be resolved in `2.5.0`.

**Comment by Kein1945** [comment]
I are realy fast with answer. Thanks for theme and work ;)
